packages/manager/apps/pci-databases-analytics/src/hooks/useHorizontalScroll.hook.tsx

```diff
@@ -0,0 +1,27 @@
+import { useEffect, useRef } from 'react';
+
+const useHorizontalScroll = () => {
+  const ref = useRef(null);
+
+  useEffect(() => {
+    const scrollContainer = ref.current;
+    if (!scrollContainer) return undefined;
+
+    const handleWheel = (e: WheelEvent) => {
+      e.preventDefault();
+      if (scrollContainer.children[1]) {
+        scrollContainer.children[1].scrollLeft += e.deltaY;
+      }
+    };
+
+    scrollContainer.addEventListener('wheel', handleWheel, { passive: false });
+
+    return () => {
+      scrollContainer.removeEventListener('wheel', handleWheel);
+    };
+  }, []);
```

Comment:
I think the useEffect should depend on the ref ? If so maybe it should be a useState instead

Reply:
Unfortunately, refs can not be used as dependencies for useEffect, as they are not reactive. I changed the logic to use a state as you suggested and it works fine, thanks 😃